sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/TagsRestOperationsTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    public class TagsRestOperationsTests : ResourceManagerTestBase
```

Comment:
We should be able to get coverage without needing a direct tagsrestoperationstests, the tests from container / operations should suffice.

Reply:
Done